### ISSUE TITLE
Using docker driver with mount flag set forces replacement 

### ISSUE THREAD

**Issue by amne** [scott-the-programmer/terraform-provider-minikube#89]
It looks like if you pass "mount = true" then it is not persisted in the state and it always forces a replacement.
This means you can't have a stack with multiple resources (like helm_release) that depend on the cluster because the cluster will be destroyed and recreated but the releases are not reinstalled since at plan time they are still installed

```terraform
resource "minikube_cluster" "docker" {
  driver       = "docker"
  cluster_name = "minikube"
  addons = [
    "default-storageclass",
    "storage-provisioner"
  ]
  mount = true
  mount_string = "/home/amne:/home/amne"
  network = "minikube"
}
```

Running `terraform plan` will output this (snipped):
` ~ mount                      = false -> true # forces replacement`

**Comment by scott-the-programmer** [comment]
Thanks reporting this! What version are you running?

I have a feeling that this is related to https://github.com/scott-the-programmer/terraform-provider-minikube/issues/86 which was fixed and released yesterday (v0.3.2)

**Comment by amne** [comment]
Indeed. I was using 0.3.1 and updated to 0.3.2 (trying to see if it fixes the network issue I had but proved to be the cni value) and now it's not happening. I will close this one also.

Thank you!
